build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -26,6 +26,18 @@
     common_definitions.CpuBenchmarkConfig(
         model=tflite_models.MOBILENET_V3SMALL, threads=[0, 1]
     ),
+    common_definitions.CpuBenchmarkConfig(
+        model=matmul.MATMUL_1x256x2048_I8xI4_MLIR, threads=[0, 1]
+    ),
+    common_definitions.CpuBenchmarkConfig(
+        model=matmul.MATMUL_256x256x2048_I8xI4_MLIR, threads=[0, 1]
+    ),
+    common_definitions.CpuBenchmarkConfig(
+        model=matmul.MATMUL_1x256x2048_I8xI8_MLIR, threads=[0, 1]
+    ),
+    common_definitions.CpuBenchmarkConfig(
+        model=matmul.MATMUL_256x256x2048_I8xI8_MLIR, threads=[0, 1]
```

Comment:
Do we need to test local-sync and 1-thread. I feel like for microbenchmark local-sync might be enough?

Reply:
+1

Reply:
Good point, I updated to use local-sync only.